src/Windows/R/Editor/Impl/Settings/REditorSettings.cs

```diff
@@ -24,15 +26,28 @@ public sealed class REditorSettings: EditorSettings, IWritableREditorSettings, I
 
         private RFormatOptions _formatOptions = new RFormatOptions();
 
-        public REditorSettings(ICoreShell coreShell): base(coreShell, RContentTypeDefinition.ContentType) { }
-        public REditorSettings(IEditorSettingsStorage storage) : base(storage) { }
+        public REditorSettings(ICoreShell coreShell) : base(coreShell, RContentTypeDefinition.ContentType) {
+            CreateLintOptions();
+        }
+
+        public REditorSettings(IEditorSettingsStorage storage) : base(storage) {
+            CreateLintOptions();
+        }
 
         public override void ResetSettings() {
             _formatOptions = new RFormatOptions();
-            LintOptions = new LintOptions();
+            CreateLintOptions();
             base.ResetSettings();
         }
 
+        private void CreateLintOptions() {
+            LintOptions = new LintOptions();
+            LintOptions.PropertyChanged += _OnLintOptionsPropertyChanged;
```

Comment:
So right now it looks like this:

| Storage | Value saved | Value loaded |
| --- | --- | --- |
| Editor  | Every time property value changed | When `LanguageSettingsStorage.Load` is called |
| DialogPage | When `SettingsHolder.Apply` is called from `RToolsOptionsPage.OnApply` | When `RToolsOptionsPage` instance is created |
| Roaming | When `VsSettingsStorage.PersistAsync` is called from `RSettingsImplementation.SaveSettingsAsync` | Every time property value requested and not cached |

Is it correct?